### PR TITLE
FLOW-74 bulk send

### PR DESCRIPTION
**Pull request by youngl-docusign**
Jira: https://docusign.atlassian.net/browse/FLOW-74

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/DocuSignDemo/script.csx [hunk 1]
```diff
@@ -2976,6 +2976,153 @@ public class Script : ScriptBase
...
+    JObject newBody = ParseCSV(body);
+    newBody["name"] = name;
+    return newBody;
+  }
+
+  private JObject ParseCSV(JObject inputBody)
+  {
+    var input = inputBody.GetValue("csv").ToString();
+    var body = new JObject();
+
```
Comment (harshitav-docusign) [inline certified-connectors/DocuSignDemo/script.csx]: Please remove extra blank lines through out this method.

File: certified-connectors/DocuSignDemo/script.csx [hunk 2]
```diff
@@ -2976,6 +2976,153 @@ public class Script : ScriptBase
...
+    try
+    {
+        var lines = input.Split(new string[] { "\r\n", "\r", "\n" }, StringSplitOptions.None);
+
+        var headerLine = lines[0];
+        var headerItems = headerLine.Split(',');
+
+        var parsedHeaders = new string[headerItems.Length][];
+
+        for (int i = 0; i < headerItems.Length; i++)
```
Comment (harshitav-docusign) [inline certified-connectors/DocuSignDemo/script.csx]: Place all declarations at the beginning of the try block. This for loop can be moved after the declarations. These small changes improve the readability of the code.

File: certified-connectors/DocuSignDemo/script.csx (hunk 1 above)
Comment (youngl-docusign) [reply certified-connectors/DocuSignDemo/script.csx]: Sure

File: certified-connectors/DocuSignDemo/script.csx (hunk 2 above)
Comment (youngl-docusign) [reply certified-connectors/DocuSignDemo/script.csx]: Sure.

Comment (youngl-docusign) [reply certified-connectors/DocuSignDemo/script.csx]: Done.

File: certified-connectors/DocuSignDemo/script.csx (hunk 1 above)
Comment (youngl-docusign) [reply certified-connectors/DocuSignDemo/script.csx]: Done

**Review by harshitav-docusign — APPROVED** [review]

**Pull request merged**